sigs.yaml

```diff
@@ -589,7 +592,7 @@ sigs:
       time: "9:30"
       tz: "PT (Pacific Time)"
       frequency: biweekly
-    meeting_url: https://zoom.us/j/852475138
+    meeting_url: https://zoom.us/my/k8s.mc
```

Comment:
Why the move back to k8s.mc? If I recall, @perotinus and @madhusudancs had identified a problem with using k8s.mc.

Reply:
There seems to be this confusion continuing from yesterday, where half of the folks seemed to be logged in to one meeting room (I assumed it was https://zoom.us/my/k8s.mc) and other half on to another (https://zoom.us/j/852475138). I tried connecting to https://zoom.us/j/852475138, which is current listed meeting room at this sig community page. While trying to connect to https://zoom.us/j/852475138, for the first 7 minutes I got an error `please wait for the host to start this meeting`, and it automatically connected me at 9:37 PT, with only 3 guys in that zoom room. All 3 of us had the same problem and assumed that this is not the correct link. Please correct me, if my assumptions are wrong. @perotinus @madhusudancs plz help.

Reply:
I believe the problem we had with using k8s.mc was enabling "join without host". We tried to fix that by moving to a custom-defined meeting instead of the personal meeting room of the SIG multicluster Zoom account, which had the meeting ID https://zoom.us/j/852475138.

I think the Google Calendar invite is out-of-date and still references the old meeting. @csbell, can you fix that?

Reply:
@csbell @perotinus, any updates. It seems that both https://zoom.us/j/852475138 and https://zoom.us/my/k8s.mc work just fine and we have used both in the recent sig or federation WG syncs. I tried both before writing this comment and could login to both the meeting ids. Please confirm choosing one of these. Meanwhile https://github.com/kubernetes/community/pull/1533 is already merged. I suggest, we continue with https://zoom.us/my/k8s.mc to avoid further reader confusion and ensure that the same is updated at all relevant places.

Reply:
@irfanurrehman @csbell Since https://zoom.us/my/k8s.mc is working fine, and it is what seems to be currently in the meeting invites and here, I agree that we should stick with it. I've deleted the other meeting from the SIG Zoom account to hopefully limit future confusion.

Reply:
thanks @perotinus for having a look at this.